R/tm_g_bivariate.R

```diff
@@ -797,8 +797,8 @@ substitute_q <- function(x, env) {
 #' bivariate_ggplot_call("numeric", "factor")
 #' bivariate_ggplot_call("factor", "numeric")
 #' bivariate_ggplot_call("factor", "factor")
-bivariate_ggplot_call <- function(x_class = c("NULL", "numeric", "integer", "factor", "character", "logical"),
-                                  y_class = c("NULL", "numeric", "integer", "factor", "character", "logical"),
+bivariate_ggplot_call <- function(x_class = c("NULL", "numeric", "integer", "factor", "ordered", "character", "logical"), # nolint: line_length
+                                  y_class = c("NULL", "numeric", "integer", "factor", "ordered", "character", "logical"), # nolint: line_length
```

Comment:
```suggestion
                                  y_class = c("NULL", "numeric", "integer", "factor", "ordered", "character", "logical"), # nolint: line_length.
```

Reply:
How about removing the default value of `x_class` and having this inside?
```
x_class <- switch(
  x_class,
  "integer" =, "numeric" = "numeric",
  "logical" =, "character" =, "factor" =, "ordered" = "factor",
  stop("unsupported x_class: ", toString(x_class))
)
```
If you add `if (is.null(x_class) && is.null(y_class)) stop("either x or y is required")`, everything will be covered.

Reply:
I like this approach 👍🏽 Much better than `match.args`